code/game/machinery/lightswitch.dm

```diff
@@ -48,6 +48,7 @@
 	area.power_change()
 
 /obj/machinery/light_switch/power_change()
+	SHOULD_CALL_PARENT(0)

```

Comment:
should be unnecessary because the proc contains a `..()` call somewhere within it